tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,194 @@
+    with mlflow.start_run() as manual_run:
+        mlflow.paddle.autolog()
+
+        paddle.set_default_dtype("float32")
+        train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+        eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+
+        class UCIHousing(paddle.nn.Layer):
+            def __init__(self):
+                super(UCIHousing, self).__init__()
+                self.fc = paddle.nn.Linear(13, 1)
+
+            def forward(self, feature):  # pylint: disable=arguments-differ
+                pred = self.fc(feature)
+                return pred
+
+        model = paddle.Model(UCIHousing())
+        optim = paddle.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
+        model.prepare(optim, paddle.nn.MSELoss())
+
+        model.fit(train_dataset, eval_dataset, epochs=NUM_EPOCHS, batch_size=8, verbose=1)
+        assert mlflow.active_run() is not None
+        assert mlflow.active_run().info.run_id == manual_run.info.run_id
+
+
+def test_paddle_autolog_ends_auto_created_run(paddle_model):  # pylint: disable=unused-argument
+    assert mlflow.active_run() is None
+
+
+@pytest.fixture
+def paddle_model_with_early_stopping(patience):
+
+    paddle.set_default_dtype("float32")
+    train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+    eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+
+    class UCIHousing(paddle.nn.Layer):
+        def __init__(self):
+            super(UCIHousing, self).__init__()
+            self.fc = paddle.nn.Linear(13, 1)
+
+        def forward(self, feature):  # pylint: disable=arguments-differ
+            pred = self.fc(feature)
+            return pred
+
+    model = paddle.Model(UCIHousing())
+    optim = paddle.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
+    model.prepare(optim, paddle.nn.MSELoss())
+
+    callbacks_earlystopping = paddle.callbacks.EarlyStopping(
+        "loss",
+        mode="min",
+        patience=patience,
+        verbose=2,
+        min_delta=0,
+        baseline=None,
+        save_best_model=True,
+    )
+
+    mlflow.paddle.autolog()
+    model.fit(
+        train_dataset,
+        eval_dataset,
+        epochs=NUM_EPOCHS,
+        batch_size=8,
+        verbose=1,
+        callbacks=[callbacks_earlystopping],
+    )
+
+    client = mlflow.tracking.MlflowClient()
+    run = client.get_run(client.list_run_infos(experiment_id="0")[0].run_id)
+    return model, run
+
+
+@pytest.mark.parametrize("patience", [1])
+def test_paddle_early_stop_params_logged(paddle_model_with_early_stopping, patience):
+    _, run = paddle_model_with_early_stopping
+    data = run.data
+    assert "monitor" in data.params
+    assert "patience" in data.params
+    assert float(data.params["patience"]) == patience
+    assert "min_delta" in data.params
+    assert "stopped_epoch" in data.params
+
+
+def test_paddle_autolog_non_early_stop_callback_does_not_log(paddle_model):
```

Comment:
Can you check the test names? For example, the name of this test doesn't match what's checked in this test?